### DISCUSSION TITLE
Adding scripts for facebook pixel, linkedin insights and google analytics

### DISCUSSION THREAD

**Discussion by xKr9** [nuxt/nuxt#16101] [Questions]
Hey all, I've been stuck on this problem for a while and I haven't had much luck solving it. I tried different approaches but the result I get is the following for each script

```Uncaught SyntaxError: Unexpected token '<' (at facebook-pixel.js:1```

These are the same scripts I've used before, so I'm under assumption its the way they are being imported. I've tried the following and always had the same result

- I've tried adding the scripts inside of the ```nuxt.config``` head
- Adding the scripts inside of the useHead composable
- Adding the scripts inside of the ```head()``` method
- Loading the script inside of the Script component

Has anyone had similar issues? Thanks to anyone replying

**Comment by zernonia** [comment]
This error is being thrown from `facebook-pixel.js`, and might be not related to Nuxt. Perhaps you are using the wrong js script?
